modules/arch-test/src/test/java/org/apache/ignite/internal/SqlEngineModuleArchTest.java

```diff
@@ -48,10 +48,10 @@ public class SqlEngineModuleArchTest {
             .should(new ArchCondition<>("override org.apache.calcite.rel.RelNode.getRelTypeName") {
                 @Override
                 public void check(JavaClass javaClass, ConditionEvents conditionEvents) {
-                    boolean overrided = javaClass.getMethods().stream()
-                            .anyMatch(it -> "getRelTypeName".equals(it.getName()));
+                    boolean methodNotImplemented = javaClass.getMethods().stream()
+                            .noneMatch(it -> "getRelTypeName".equals(it.getName()));
 
-                    if (!overrided) {
+                    if (methodNotImplemented) {
```

Comment:
If someone will rollback current PR due to any reason, this fix will be lost as well.
You can left this code in patch/comment to the followup ticket.

Reply:
Thanks, changes reverted.